english/bob-barker/barker.md

```diff
@@ -1,8 +1,10 @@
 The door thuds open and the light coming from within the room in front of you blinds you momentarily.
 
-When your vision clears, standing before you is Bob Barker in all his glory.  
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.